src/main/java/wolox/training/exceptions/RestExceptionHandler.java

```diff
@@ -0,0 +1,45 @@
+package wolox.training.exceptions;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import javax.validation.ConstraintViolationException;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler({BookNotFoundException.class})
+    protected ResponseEntity<Object> handleNotFound(Exception ex, WebRequest request) {
+        return handleExceptionInternal(ex, "Entity not found", new HttpHeaders(),
```

Comment:
We need a way to not put literals message errors in controllers, whe need a way to keep message errors all together and separated.